live/engine/venue.py

```
@@ -0,0 +1,141 @@
+        return None
+    return ApiCreds(api_key=api_key, api_secret=secret, api_passphrase=passphrase)
+
+
+def open_notional(c) -> float:
+    """Dollars currently committed in open resting orders on the venue."""
+    try:
+        orders = c.get_open_orders() or []
+        return sum(float(o.get("price", 0) or 0)
+                   * float(o.get("original_size", 0) or 0)
+                   for o in orders)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)